content/docs/1.0.x/install/cli-install/index.md

```diff
@@ -4,68 +4,125 @@ description: Install binaries for the Keptn CLI
 weight: 35
 ---
 
-Install the Keptn CLI on the local machine
-that is used to interact with your cloud provider, Kubernetes, etc.
 The Keptn CLI is not Keptn itself
 but is instead an interface to Keptn,
 much as **kubectl** is an interface to Kubernetes.
+Install it on the local machine
+that is used to interact with your cloud provider, Kubernetes, etc.
 
 The Keptn CLI sends commands to Keptn by interacting with the Keptn API.
 The [API Token](../../operate/api_token/)
 that is used to communicate with Keptn is generated during the installation.
 
-It is not absolutely necessary to install the Keptn CLI for current Keptn releases,
-which allow you to create a project or run a sequence using the [Keptn Bridge](../../bridge/),
-but it is recommended for production systems.
+It is possible to run Keptn without installing the Keptn CLI.
+Current Keptn releases allow you to create a project or run a sequence using the [Keptn Bridge](../../bridge/),
+and this may be adequate for demos or personal environments you set up for study..
+But the CLI is required for some functionality
+and it should be installed on production systems.
 
 You can install the Keptn CLI using:
 
 * curl
 * Homebrew (MacOS only)
 * Standard binary installation tools for your operating system
 
-## curl
+## Install with curl
 
-```
-curl -sL https://get.keptn.sh/ | bash
-```
+You can use the `curl` command to install the Keptn CLI on Linux, WSL2, and macOS.
+Windows users need `bash`, `curl`, and `awk` installed (e.g., using Git Bash).
 
-This installs the Keptn CLI for you in the current directory on your cloud shell machine.
-You can use **sudo** to move the file to another location.
-You can run the Keptn CLI from any directory
-but you may need to specify the location of the path,
-using `.keptn` rather than just `keptn` if it is in your current directory.
+1. Download the *latest stable Keptn version*
+   from [GitHub](https://github.com/keptn/keptn/releases/),
+   unpack it, and install it into the current directory on your cloud shell machine:
 
-## Homebrew
+   ```
+   curl -sL https://get.keptn.sh/ | bash
+   ```
 
-```
-brew install keptn
-```
+   Use **sudo** to move the file to another location.
+   such as `/usr/local/bin/keptn`.
+   You can run the Keptn CLI from any directory
+   but you may need to specify the location of the executable
+   or add the directory to your $PATH definition.,
+   Remember to use  `./keptn` rather than just `keptn` if it is in your current directory.
```

Comment:
Is this really information that we need to provide in our docs? IMHO this is general info that we don't need provide again...

Reply:
Yeah, I wondered why we had this little foray into *nix shell basics but it was in the old docs so I moved it over.  It.s gone now.